paderbox/transform/module_resample.py

```diff
@@ -50,11 +50,14 @@ def resample_sox(signal: np.ndarray, *, in_rate, out_rate):
         signal: Signal as one-dimensional np.ndarray: Shape (T,)
         in_rate: Probably as an integer
         out_rate: Probably as an integer
+        normalize: If True, normalize input to be between -1 and 1, befor
+            calling sox. After resampling, the normalization is reverted.
+            Sox has the issue, to clip some values, if they are above 1.
 
     Returns: Resampled version with same dtype as input.
 
     """
-    assert signal.dtype == np.float32, (
+    assert signal.dtype in [np.float32, np.float64], (
```

Comment:
The error message sounds strange to me and has float32 hard-coded. Maybe write "The call to SOX just supports float32 and float64, but signal.dtype={signal.dtype}."

Reply:
Done